app/client/src/components/pages/LocationMap/mapStyles.css

```diff
@@ -147,7 +147,7 @@
   box-shadow: 0 1px 2px rgba(0, 0, 0, 0.3);
 }
 
-.hmw-map-toggle p {
+.hmw-map-toggle h4 {
```

Comment:
Can you revert this back to `p` as well?

Reply:
@cschwinderg ran prettier again and reverted that selector back to a `p` tag. Do you need me to change the `h4` back to a `p` anywhere else?

Reply:
@oftheheadland Looks good! Thank you for making those changes.